bindings/aws/s3/metadata.yaml

```diff
@@ -0,0 +1,122 @@
+# yaml-language-server: $schema=../../../component-metadata-schema.json
+schemaVersion: v1
+type: bindings
+name: aws.s3
+version: v1
+status: stable
+title: "AWS S3 Bindings"
+urls:
+  - title: Reference
+    url: https://docs.dapr.io/reference/components-reference/supported-bindings/s3/
+binding:
+  output: true
+  operations:
+    - name: create
+      description: "Create blob."
+    - name: get
+      description: "Get blob."
+    - name: delete
+      description: "Delete blob."
+    - name: list
+      description: "List blob."
+capabilities: []
+builtinAuthenticationProfiles:
+  - name: "aws"
+    metadata:
+    - name: accessKey
+      type: string
+      required: true
+      sensitive: true
+      description: |
+        AWS access key associated with an IAM account.
+      example: "AKIAIOSFODNN7EXAMPLE"
+      default: ""
+    - name: secretKey
+      type: string
+      required: true
+      sensitive: true
+      description: |
+        The secret key associated with the access key.
+      example: "wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY"
+      default: ""
```

Comment:
These are included in the built-in authentication profile so need to be removed from here

```suggestion
```